README.md

```diff
@@ -63,5 +63,12 @@ In addition to setting a default Cloud Run region, make sure to deploy to the fu
 can achieve this by setting the `run/platform` gcloud property to `managed` or passing in the `--platform=managed` flag
 to your `gcloud run` commands.
 
+### Defaults
+
 If comment code tags aren't added to your README, the program will fall back to reasonable defaults to build and deploy
-your sample to Cloud Run based on whether your sample is java-based and doesn't have a Dockerfile or isn't.
+your sample to Cloud Run based on whether your sample is Java-based (has a pom.xml) or not.
```

Comment:
See my other comments about having this be a list

Reply:
Resolved, thank you.